### ISSUE TITLE
Send Direct Message Form - Loads at bottom of page

### ISSUE THREAD

**Issue by s38b35M5** [PrivacyDragon/Bookwyrm_Android#6]
Version: 1.3.4

Steps to reproduce: Tap three dot menu on user profile > send direct message 

Result: Loads top of feed page with message form at bottom, giving impression message form didn't load

Desired Result: Should bring user to the compose form to send a message

**Comment by PrivacyDragon** [comment]
I am unable to reproduce this, but I think that I know what you mean.
It might indeed look like the feed page is loaded, but when you scroll down, you should see the compose form.

I will try to see if I can add in that the app automatically scrolls down. If possible.

**Comment by s38b35M5** [comment]
You're absolutely right! I didn't know to scroll down! Perhaps a scroll would be helpful for others, but feel free to close as it's "as designed" if the scroll fix isn't as easy as you hope.
